.circleci/config.yml

```diff
@@ -68,14 +60,18 @@ jobs:
     docker:
       - image: node:10
 
+  build-node_11:
+    <<: *latest-build
```

Comment:
Sorry. Can you change `latest-build` to `common-build` in `build-node_10` line 59 so we don't push coverage twice?

Reply:
Sorry have never dealt with this file / config before -- should have been less cavalier when copy-pasting the last command (node 10 -> 11) 😓